engine/sound/src/decoders/decoder_tremolo.cpp

```diff
@@ -149,22 +149,33 @@ namespace dmSoundCodec
         return (got_bytes == 0 && streamInfo->m_bEOS) ? RESULT_END_OF_STREAM : RESULT_OK;
     }
 
+    // Called both when looping, and when deleting the decoder (!)
```

Comment:
The following parts are part of the previous PR (https://github.com/defold/defold/pull/10019). I just wanted to avoid crashing the engine while testing.